src/test/java/io/cdap/plugin/sfmc/connector/MarketingConnectorTest.java

```diff
@@ -100,8 +117,7 @@ private Schema getPluginSchema() throws IOException {
 
   @Test
   public void testConvertRecord() throws ETSdkException, NoSuchFieldException {
-    MarketingConnectorConfig marketingConnectorConfig1 = new MarketingConnectorConfig
-            ("clientId", "clientSecret", "authEndpoint", "soapEndpoint");
+    MarketingCloudConversion cloudConversion = new MarketingCloudConversion();
```

Comment:
MarketingCloudConversion -> MarketingCloudUtil

Reply:
Changed it to MarketingCloudUtil